src/components/Shared/TableStickyColumns.scss

```
@@ -0,0 +1,44 @@
+/**
+* Utility class for tables with 2 sticky left columns.
+* Works with Semantic UI `.ui.celled.table` produced by QHGrid.
+*
+* Usage:
+*   <div className="two-col-sticky">...</div>
+*   <div id="DifferentialResultsTableWrapperCheckboxes" className="two-col-sticky">...</div>
+*/
+.two-col-sticky {
+  --frozen-first-column-width: 35px;
+
+  .ui.celled.table {
+    position: relative;
+    border-collapse: separate;
+    border-spacing: 0;
+  }
+
+  .ui.celled.table :is(th, td):nth-child(-n + 2) {
+    position: sticky;
+    background-color: #ffffff;
+    z-index: 1;
+  }
+
+  /* First column sticks at the very left */
+  .ui.celled.table thead tr th:nth-child(1),
+  .ui.celled.table tbody tr td:nth-child(1) {
```

you should be able to just use `:is` here to make this much simpler.

```suggestion
  .ui.celled.table :is(thead,tbody) tr :is(th,td):nth-child(1){
```